recipe/meta.yaml

```diff
@@ -1,38 +1,49 @@
 {% set name = "dash_cytoscape" %}
-{% set version = "0.2.0" %}
+{% set version = "0.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 0669c79c197e4b150a5db7a278d1c7acebc947f3f5cbad5274835ebb44f712cd
+  sha256: a71ad4fe095570b71d4ad7c0d29199e9780c2e6796173d3b25fccc2cc58c855f
+  patches:
+    - 0001-point-to-correct-file-location.patch
 
 build:
-  noarch: python
-  number: 1
-  script: '{{ PYTHON }} -m pip install . -vv '
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
+  build:
+    - patch  # [unix]
+    - m2-patch  # [win]
   host:
     - pip
-    - python >=3.5
+    - setuptools
+    - wheel
+    - python
   run:
     - dash
```

Comment:
dash >= 1.0 , <2.0 
https://github.com/plotly/dash-cytoscape/blob/3799bbdfbc2b631a7cea98de67a7c5eeacc8ee78/requirements.txt#L5
?

Reply:
I am not sure. setup.py does not use requirements.txt file. In setup.py, only unpinned dash is mentioned:

https://github.com/plotly/dash-cytoscape/blob/v0.3.0/setup.py#L25

Reply:
In doubt, download the source from pypi. setup.py in the source only lists dash, so what Anton did seems right to me.